Resources/config/services.xml

```diff
@@ -5,36 +5,34 @@
     xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
     <services>
-        <service id="ekino.new_relic.interactor.real" class="Ekino\Bundle\NewRelicBundle\NewRelic\NewRelicInteractor" public="false" />
-        <service id="ekino.new_relic.interactor.blackhole" class="Ekino\Bundle\NewRelicBundle\NewRelic\BlackholeInteractor" public="false" />
-        <service id="ekino.new_relic.interactor.adaptive" class="Ekino\Bundle\NewRelicBundle\NewRelic\AdaptiveInteractor" public="false">
-            <argument type="service" id="ekino.new_relic.interactor.real" />
-            <argument type="service" id="ekino.new_relic.interactor.blackhole" />
+        <service id="Ekino\NewRelicBundle\NewRelic\NewRelicInteractor" public="false" />
+        <service id="Ekino\NewRelicBundle\NewRelic\BlackholeInteractor" public="false" />
+        <service id="ekino.new_relic.interactor.adaptive" class="Ekino\NewRelicBundle\NewRelic\AdaptiveInteractor" public="false">
+            <argument type="service" id="Ekino\NewRelicBundle\NewRelic\NewRelicInteractor" />
+            <argument type="service" id="Ekino\NewRelicBundle\NewRelic\BlackholeInteractor" />
         </service>
 
-        <!-- Default alias -->
-        <service id="ekino.new_relic.interactor" alias="ekino.new_relic.interactor.blackhole" />
         <!-- Help auto complete -->
-        <service id="Ekino\Bundle\NewRelicBundle\NewRelic\NewRelicInteractorInterface" alias="ekino.new_relic.interactor" />
+        <service id="Ekino\NewRelicBundle\NewRelic\NewRelicInteractorInterface" alias="ekino.new_relic.interactor" />
 
-        <service id="ekino.new_relic.interactor.logger" class="Ekino\Bundle\NewRelicBundle\NewRelic\LoggingInteractorDecorator" public="false">
-            <argument type="service" id="ekino.new_relic.interactor.real" />
+        <service id="Ekino\NewRelicBundle\NewRelic\LoggingInteractorDecorator" public="false">
+            <argument type="service" id="Ekino\NewRelicBundle\NewRelic\NewRelicInteractor" />
```

Comment:
IMHO should be NewRelicInteractorInterface here

Reply:
You are correct. This is wrong. 

I thing we should leave it empty. The extension will overwrite this value anyways, right?

Reply:
Yes it could be done by the extension too

Reply:
It is done already: https://github.com/ekino/EkinoNewRelicBundle/blob/master/DependencyInjection/EkinoNewRelicExtension.php#L53